space-invaders.asm

```diff
@@ -20,9 +20,10 @@
 %define ICON_EXPLOSION_BULLET '#'
 %define ICON_WALL '#'
 
-; GAME WINNERS
-%define PLAYER_WIN 0
-%define INVADERS_WIN 1
+; GAME STATES
+%define STILL_PLAYING 0
+%define PLAYER_WIN 1
+%define INVADERS_WIN 2
```

Comment:
These constants should also include "type" information in the name.

```asm
%define GAME_STATE_PLAYING 0
%define GAME_STATE_PLAYER_WINS 1
%define GAME_STATE_INAVERS_WIN 2
```

Reply:
Does it have the same meaning as above(https://github.com/flxbe/asm-space-invaders/pull/15#discussion_r226906744)?